src/NuGet.Core/NuGet.Configuration/Settings/Items/StoreClientCertItem.cs

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Xml.Linq;
+
+namespace NuGet.Configuration
+{
+    /// <summary>
+    ///     A StoreClientCertItem have 4 Attributes:
+    ///     - [Required] packageSource
+    ///     - [Optional] storeLocation. StoreLocation.CurrentUser by default.
+    ///     - [Optional] storeName. StoreName.My by default.
+    ///     - [Optional] findBy. X509FindType.FindByThumbprint by default.
+    ///     - [Required] findValue
+    /// </summary>
+    public sealed class StoreClientCertItem : ClientCertItem
+    {
+        private const X509FindType DefaultFindBy = X509FindType.FindByThumbprint;
+        private const StoreLocation DefaultStoreLocation = StoreLocation.CurrentUser;
+        private const StoreName DefaultStoreName = StoreName.My;
+
+        public static string GetString(X509FindType type)
+        {
+            return type.ToString().Replace("FindBy", string.Empty);
+        }
+
+        public static string GetString(StoreName storeName)
+        {
+            return storeName.ToString();
+        }
+
+        public static string GetString(StoreLocation storeLocation)
+        {
+            return storeLocation.ToString();
+        }
+
+        public StoreClientCertItem(string packageSource,
+                             string findValue,
+                             StoreLocation? storeLocation = null,
+                             StoreName? storeName = null,
+                             X509FindType? findBy = null)
+            : base(packageSource)
```

Comment:
Unusual indenting. We don't use resharper, so we normally indent 1 level (4 spaces) from the first line. We don't indent to line up to the open parenthesis.

Reply:
It is glitch after class rename. Fixed